### ISSUE TITLE
Update is broken. Freezing

### ISSUE THREAD

**Issue by Suncatcher** [osmandapp/Osmand#7036]
I want to highlight that this bug is present on all devices and all Android versions (from 5.0 to 8.0), and on different networks. It persists on last 10 updates and maybe earlier.

I cannot update local maps normally, whenever I start update process (via Osmand menu) it goes 1-10% and then freezing. Only full relaunch (cancel button) helps and even that doesn't help always. Sometimes I have to delete the full map file and re-download and sometime even reboot the device.

As a paid version user I want it to be fixed.

What version of OsmAnd are you using, on what device and Android/iOS version?
**Latest. Samsung SM-T580 tablet with Android Oreo**

Do you use the offline maps offered within the OsmAnd app for download, or online (tile / raster) maps?
**Offline**

If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs, and its edition date.
**Not related to certain map**

**Comment by vshcherb** [comment]
So far, don't remember any experiences, could it be related to sdcard degradation. Is it observed uring download of new map as well?

**Comment by Suncatcher** [comment]
>  sdcard degradation

no, it is observed on many devices and in the last case it is device memory, not sdcard.

And it is observed only on updating maps, no new, that's why I think it is flaw in update process.

**Comment by vshcherb** [comment]
I could definitely say that we are working on new download mechanics so it doesn't stop in background on some Android versions and it will be ready next weeks.
Though I wouldn't be very optimistic that it helps in your case (50/50). I'm now collecting devices and users where we can get that issue reproduced, so we can debug it. Unfortunately it is hard to assume  now what goes wrong. Anyway if it wouldn't be difficult to make a short clip it would help tremendously.
Thanks

**Comment by Suncatcher** [comment]
Video clip? Ok, will do
